steps/cli/oc.go

```diff
@@ -165,6 +166,38 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify versioned ecosystem tasks", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	// Get the arch of the cluster as kn and Kn-apply task are not available on arm64 cluster
+	getClusterArch := cmd.MustSucceed("oc", "get", "node", "-o", "jsonpath='{.items[0].status.nodeInfo.architecture}'").Stdout()
+	clusterArch := strings.TrimSpace(getClusterArch.string())
+	requiredTasks := []string{"buildah", "git-cli", "git-clone", "kn", "kn-apply", "maven", "openshift-client", "s2i-dotnet", "s2i-go", "s2i-java", "s2i-nodejs", "s2i-perl", "s2i-php", "s2i-python", "s2i-ruby", "skopeo-copy", "tkn"}
+	getRequiredVersion := os.Getenv("OSP_VERSION")
+	if clusterArch == "arm64" {
+		requiredTasks = filterTasks(requiredTasks, []string{"kn", "kn-apply"})
```

Comment:
What is `filterTasks`? The code is not compilable.

Reply:
Fixed